src/components/Badge.test.tsx

```diff
@@ -5,19 +5,34 @@ import { pathwayTypeTooltips, sectorTooltips } from "../utils/tooltipUtils";
 
 describe("Badge component", () => {
   it("renders with the provided text", () => {
-    render(<Badge text="Test Badge" />);
+    render(<Badge>Test Badge</Badge>);
     expect(screen.getByText("Test Badge")).toBeInTheDocument();
   });
 
+  it("renders numbers as content", () => {
+    render(<Badge>{123}</Badge>);
```

Comment:
Silly question, but why the need for the squiggly braces?

Reply:
JSX: anything in braces is interpolated JS code (this is why all the variable names go in braces). Anything that isn't is just plain HTML(ish) strings.

So we need the `{123}` to get it to render a `number` type. `<Badge>123</Badge>` would be a string, same as `<Badge>{"123"}</Badge>`